docs/docsite/rst/user_guide/intro_inventory.rst

```diff
@@ -304,7 +304,7 @@ is an excellent way to track changes to your inventory and host variables.
 How Variables Are Merged
 ++++++++++++++++++++++++
 
-By default variables are merged/flattened to the specific host before a play is run. This keeps Ansible focused on the Host and Task, so groups don't really survive outside of inventory and host matching. By default, Ansible overwrites variables including the ones defined for a group and/or host (see the `hash_merge` setting to change this) . The order/precedence is (from lowest to highest):
+By default variables are merged/flattened to the specific host before a play is run. This keeps Ansible focused on the Host and Task, so groups don't really survive outside of inventory and host matching. By default, Ansible overwrites variables including the ones defined for a group and/or host (see :ref:`ANSIBLE_HASH_BEHAVIOUR<envvar-ANSIBLE_HASH_BEHAVIOUR>`). The order/precedence is (from lowest to highest):
```

Comment:
don't reference the env var, that is same limitation as using just the ini, point at the base config option, which contains both

Reply:
we want users to end up here: https://docs.ansible.com/ansible/latest/reference_appendices/config.html#default-hash-behaviour

better than here:
https://docs.ansible.com/ansible/latest/reference_appendices/config.html#envvar-ANSIBLE_HASH_BEHAVIOUR

Reply:
Lemme redo and issue again. Thanks @bcoca